### PR TITLE
BEE-49248: Process reservations under project

### PR DESCRIPTION
**Pull request by cbameyer**
preflight: https://artemis.nimbus.beescloud.com/flow/#/pipeline-run/d56c2bcf-3013-11eb-853c-42010a1eb00c/1c6bb7cb-3dde-11ef-afbd-42010a1eb05e

### CODE REVIEW

**Comment by cbameyer** [comment]
@urvashisingh confirmed that we want to add a note regarding non-backward compatibility in the doc here (and have an error message in nimbus).

File: help/help.adoc [hunk 1]
```diff
@@ -368,6 +368,10 @@ NOTE: If the Git commit ID cannot be found or if the procedure or steps cannot f
 [[releaseNotes]]
 == DslDeploy plugin release notes
 
+=== 4.3.2
+
+* Process reservations under project. (Note that previously exported DSLs will need to be re-exported for import to succeed)
```
Comment (jsmith-cb) [inline help/help.adoc:373]:
```suggestion
* Process reservations under project. 
+
NOTE: Previously exported DSLs must be re-exported for import to succeed.
```

**Comment by karakurty** [comment]
@cbameyer could you please add test coverage.

**Review by jsmith-cb — APPROVED** [review]

**Comment by cbameyer** [comment]
@karakurty I have modified the tests to ensure process reservation under project is successful.

Does that work?

**Review by karakurty — APPROVED** [review]

**Pull request merged**